gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/proc/DeadlineEnforcementDagProc.java

```diff
@@ -68,7 +68,7 @@ protected boolean isDagStillPresent(Optional<Dag<JobExecutionPlan>> dag, DagMana
 
     if (!dagManagementStateStore.existsJobDagAction(dagAction.getFlowGroup(), dagAction.getFlowName(),
         dagAction.getFlowExecutionId(), dagAction.getJobName(), dagAction.getDagActionType())) {
-      log.warn("Dag action {} is cleaned up from DMSS. No further action is required.", dagAction);
+      log.info("Dag action {} is cleaned up from DMSS. No further action is required.", dagAction);
       return false;
```

Comment:
don't we already hold a lease by the time `act` is called?  if so, it would be exceedingly rare (and really no different than any other `DagProc::act` derived class impl).  hence, I'd suggest to remove.  but if we really believe we need this, let's put the impl into the `DagProc` base class